### DISCUSSION TITLE
AI 在解決「開放性問題」時，比人類更有創造力

### DISCUSSION THREAD

**Discussion by OG-Matcha** [1101-GS4719-C/Python-Community-GS4719-C#1908] [科技新知]
我在看新研究技術的時候發現了這篇發表在 Scientific Reports 的[研究](https://www.nature.com/articles/s41598-024-53303-w?utm_source=ai.briefnewsletter.com&utm_medium=newsletter&utm_campaign=microsoft-ai)，內容提出 OpenAI 的 GPT-4 在解決「開放性
問題」、「尋找創新解答」的能力，已經超越了人類。

![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/85878531/87cae008-189a-4ce6-a7ae-a14b6f90a925)

我認為這個研究結果可以很大程度去弭補一個人想法被侷限住的盲點。

研究團隊設計了三大類的測試，並將 GPT-4 與 151 名實驗參與者的表現進行比較，測試包含：

1. 尋找日常物品，例如：繩子或叉子，還有什麼其他用途。
2. 想象如果發生某些假設情況會有什麼後果。
3. 找出意思差很遠的名詞。

研究結果發現，GPT-4 在所有測試項目中都比人類表現更好，但由於 AI 缺乏「主動性」，只有在人類的引導下，才能發揮這種潛力。

在我看來，我覺得這顯示了GPT-4的創新能力，它可以用來提供更多的啟發或是理解抽象概念的延伸，然而，由於GPT-4的侷限性，我們如何去提供輸入和目標、如何與 GPT-4 進行有效的互動和合作才是最重要的，以上是我的見解。

**Comment by benbebobo149** [comment]
我對這篇文章有些想法。我反而比較好奇，你或是研究中對於創新的定義是甚麼，這個想法就像是在早期，人們害怕AI將會突發奇想，導致人類滅亡。殊不知，這些人都是不了解AI的訓練方式，很多東西只是AI拿去排列組合，並不是它真的無中生有創造一件事情。試想如果是在遠古時代，沒有經過訓練的AI會鑽木取火嗎?不可能，人類的可能性還是比較高。最後，我要感謝同學分享這個讓我思考的話題。

**Comment by SusuTima** [comment]
很多人都害怕在未來人類會被AI取代，我的認為是：AI存在的意義是用來幫助人類、輔助人類，而不是存在來取代人類的，AI會在人類的引導下產生更多功用。比較害怕的反而是有心人士拿AI來做為非作歹的事。

**Comment by sunset513** [comment]
我贊同樓主說的 ai 缺乏主動性，這意味著它需要人類的引導才能發揮其潛力。這使得如何提供輸入和目標、以及如何與 GPT-4 進行有效的互動和合作成為了關鍵問題。

許多人害怕 ai 取代人類，但我不支持這個觀點。我認為 ai 只是你的副駕駛，透過有創造性和實用性的想法，不只可以提升效率，還有可能將你自身的優勢放大。就像是我會透過 ai 輔助我圖文創作，它能提升我的工作效率，也可以給我很多意想不到的想法。感謝這位同學拋出這個有意義的話題。

**Comment by KH1126** [comment]
AI缺乏主動性，需要人類的引導才能充分發揮其潛力。
這意味着知道如何提供輸入和判斷最後目標至關重要，
我認為如何與GPT-4進行更好的互動，
會成爲是否可以增加新創想法的關鍵。

**Comment by Eileen050824** [comment]
AI身為機器，對指令的敏感度以及解讀的精確度毋庸置疑，但如同文中所述，如果沒有我們的開頭去帶出AI的舉動，所有的創意也都僅限在指令和資料庫中。我覺得人類比起追求勝過AI的能力，更應該思考的是我們怎麼做為那個AI引路人，上下的配合才是對我們最好的策略！

**Comment by YifenLin** [comment]
我認為研究結果中所謂的「表現好」應該受到更多關注，因為這可能取決於測試的設計以及定義。雖然GPT-4在各項測試中表現優異，但我們應該注意到它的「主動性」依賴於人類的引導。這個發現突顯了GPT-4的創新能力，但我們也必須探討如何最有效地與它合作，以及如何確保我們的輸入和目標對其有效。

**Comment by wanka2030** [comment]
當有引導的帶入時，人類的思考應該會被侷限化，而AI卻能擁有更多的想法。例如：今天要蓋一個遊樂園，你可能會依照你去過的遊樂園或者你熟知的遊樂園作為發想，但AI卻能倚靠的不僅是前者所有用的資料，更多的可能是有關聯性的題材，從遊樂園發想到游泳圈都是充滿可能性的。但人的創意正是因為被受限，才有可能會激發出更好的創意想法（正為了企劃案的創意而苦惱😖

**Comment by EPHMR0524** [comment]
在幾十年前AI還是一個做為論文題目會被學界唾棄的主題，但是隨著硬體的更新以及新的演算法以及開發人員的持續貢獻，使得AI發展得如此強大，甚至在部分領域有不遜於專家的表現，也因此許多人會擔心未來會找不到工作，位子都被AI取代，但是要說全面取代人類的話我覺得也不是這麼簡單的事，因為現在AI能發展是因為有開發人員一直給予相關資料以及引導式的問題，AI才能做到看似創新的思維，人類如果沒有給適合的暗示，AI也不一定能做到比人類還有想像力，要使得AI能完全獨立思考，還是有一段不小的距離

**Comment by wanderer657** [comment]
OpenAI的GPT-4在解決開放性問題方面超越了人類。相比其他應用，GPT-4提供更多的啟發和理解抽象概念的能力。然而，我覺得它缺乏主動性，需要人類的引導。這突顯了GPT-4的創新性，但我認為仍需要解決如何提供有效輸入和與之互動的挑戰。

**Comment by YANRONG0123** [comment]
再怎麼樣有創造性，AI所輸入進去的資源，其所代表的基礎，也是前人累積下來的，至於創新的部分，我覺得AI還是沒有比較好，我認為AI現階段只能當成一種工具，與人類互動性需要多加提升。

**Comment by yxin214** [comment]
以前總覺得就算人工智慧再厲害也很難取代人類，因為他們缺乏想像力及創造力。然而，此研究卻顯示了人工智慧在某些情況下能比人類更具創造力。儘管如此，我並不擔心人類就此被取代，如同文章所述，人工智慧需被引導才能發揮這項淺力，因此他將成為人們很好的輔助工具。即使現今有許多工作逐漸被新科技取代，如點餐人員被點餐機取代等，也有許多新興工作因人工智慧而起，我們應用正面態度看待科技的進步。

**Comment by LinChloe** [comment]
我自己想了下這些問題:
1.尋找日常物品，例如：繩子或叉子，還有什麼其他用途。
我平常找東西都要很久。
2.想象如果發生某些假設情況會有什麼後果。
是要深度思考(時間)還是廣度思考(種類)
但比起這些我更喜歡來個火箭炸飛地球。
3.找出意思差很遠的名詞。
原本想是反義詞，但發現它是名詞之後，好吧，你贏了。
突然好奇，護手霜的相反名詞是什麼?
有形的東西是不是對應到無形的。
黏黏的東西是對應到粗糙的嗎?
要怎麼評分意思相反。
噪音和閃電，哪個和護手霜的意義差最多?
